src/CheckoutSdk/Payments/Hosted/HostedPaymentRequest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Payments.Hosted
+{
+	public sealed class HostedPaymentRequest : IEquatable<HostedPaymentRequest>
+	{
+		public long? Amount { get; set; }
+
+		public Currency? Currency { get; set; }
+
+		public string Reference { get; set; }
+
+		public string Description { get; set; }
+
+		public CustomerRequest Customer { get; set; }
+
+		public ShippingDetails ShippingDetails { get; set; }
+
+		public BillingInformation Billing { get; set; }
+
+		public PaymentRecipient Recipient { get; set; }
+
+		public ProcessingSettings Processing { get; set; }
+
+		public IList<Product> Products { get; set; }
+
+		public RiskRequest Risk { get; set; }
+
+		[JsonProperty(PropertyName = "success_url")]
```

Comment:
The enforce serialization in net project is not needed. Unless there is a different name than underscore.

Reply:
@Armando-Ibarra and @rsaestrela Should I remove enforce serialization for success_url, cancel_url, failure_url, 3ds and capture_on?

Reply:
By default the serializer converts from `SuccessUrl` to `success_url` (camel case to snake case). In this case is not needed. We are doing it in Java just be more informative. It's ok to have it, but it's not necessary.

It's only strictly mandatory in cases like this #116